src/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -270,5 +276,22 @@ private void VerifyAuthorizationResult()
                 throw new MsalServiceException(_authorizationResult.Error, _authorizationResult.ErrorDescription, null);
             }
         }
+
+        private async Task CheckForBrokerAndAcquireAuthorizationAsync(CancellationToken cancellationToken)
+        {
+            Broker = brokerFactory.CreateBrokerFacade(ServiceBundle);
+
+            if (Broker.CanInvokeBroker(_interactiveParameters.UiParent))
+            {
+                _msalTokenResponse = await Broker.AcquireTokenUsingBrokerAsync(
+                    _authenticationRequestParameters.CreateRequestParametersForBroker()).ConfigureAwait(false);
+            }
```

Comment:
this should only be done if BrokerEnabled, right? Please check that first before creating the broker object